server/etcdserver/api/etcdhttp/health_test.go

```diff
@@ -323,3 +328,66 @@ func checkHttpResponse(t *testing.T, ts *httptest.Server, url string, expectStat
 		}
 	}
 }
+
+func checkMetrics(t *testing.T, path, checkName string, expectStatusCode int) {
+	rootGaugeVal := ptu.ToFloat64(rootHealthCheckGauge.With(prometheus.Labels{
+		"type": path,
+	}))
```

Comment:
Is it possible to get the real value from `prometheus.DefaultRegisterer`?

For example: `prometheus.DefaultRegisterer.Gather()`

Reply:
what do you mean by real value? These are not mocks.

Reply:
updated.